spec/tools/environment_builders/openstack/tests/interaction_methods_spec.rb

```diff
@@ -1,6 +1,6 @@
 require_relative '../interaction_methods'
 
-$LOAD_PATH << Rails.root.join("tools")
+$LOAD_PATH << Rails.root.join("tools").to_s
```

Comment:
I think these changes are nice.
nice to be consistent and avoid mistakes due to copy/pasting.

Reply:
Yeah, I didn't think about it but yes, this leak affects tests too with not only our autoload_paths but these too.  Nice find.

Reply:
Well, I more had to do this because it sporadically caused the tests to fail.  See the commit message on the commit that adds this.

Reply:
Nah, you cleverly found this on your own cuz you're awesome.  Don't be lying 😉

Reply:
Have to be right on the internet:

https://travis-ci.org/ManageIQ/manageiq/jobs/330569116#L1245-L1247